doc/visual-programming/source/widgets/data/preprocess.rst

```diff
@@ -29,33 +29,35 @@ Description
 
 Preprocessing is crucial for achieving better-quality analysis results.
 The **Preprocess** widget offers five preprocessing methods to improve
-data quality. In this widget you can immediately discretize continuous
+data quality. In this widget, you can immediately discretize continuous
 values or continuize discrete ones, impute missing values, select
 relevant features or center and scale them. Basically, this widget
 combines four separate widgets for simpler processing.
 
 .. figure:: images/preprocess-stamped.png
 
-1. List of preprocessors
+1. List of preprocessors. You drag the preprocessors you wish to use to the left side of the widget. 
 2. Discretization of continuous values
 3. Continuization of discrete values
-4. Impute missing values or remove them
+4. Impute missing values or remove them.
 5. Select the most relevant features by information gain, gain ratio,
-   Gini index
-6. Centering and scaling of features
-7. Shuffling of features.
-8. When the box is ticked (*Auto commit is on*), the widget will
-   communicate the changes automatically. Alternatively, click *Commit*.
+   Gini index.
+6. Select random features
+7. Normalize features
+8. Randomize
+9. When the box is ticked (*Send Automatically*), the widget will
+   communicate changes automatically. Alternatively, click *Send*.
+10. Produce a report. 
 
 Example
 -------
 
-In the example below we have used *adult* data set and preprocessed the
+In the example below, we have used the *Adult* data set and preprocessed the
```

Comment:
> the *Adult* data set

Is this intentionally capitalized? I'd stick to lowercase as we have it in our data sets...